### ISSUE TITLE
Redis - Pipelining Deprecation Message

### ISSUE THREAD

**Issue by simonrentzke** [kenaniah/sidekiq-status#11]
Pipelining commands on a Redis instance is deprecated and will be removed in Redis 5.0.0.

```ruby
redis.multi do
  redis.get("key")
end
```

should be replaced by

```ruby
redis.multi do |pipeline|
  pipeline.get("key")
end
```

(called from /Users/xxx/.gem/ruby/3.0.1/gems/sidekiq-status-2.1.2/lib/sidekiq-status/storage.rb:16:in `block in store_for_id'}

**Comment by blackst0ne** [comment]
PR #12

**Comment by kenaniah** [comment]
Released in v2.1.3
